packages/database/src/migrations/20210112015125-AddHeatmapContactsConfirmedSuspectedCasesSCovidSamoa-modifies-data.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+async function deleteMapOverlay(db, mapOverlayId) {
+  return db.runSql(`
+    DELETE FROM
+      "mapOverlay"
+    WHERE
+      "id" = '${mapOverlayId}';
+  `);
+}
+
+async function deleteMapOverlayInMapOverlayGroupRelation(db, childId) {
+  return db.runSql(`
+    DELETE FROM
+      "map_overlay_group_relation"
+    WHERE
+      "child_id" = '${childId}';
+  `);
+}
+
+const mapOverlay = {
+  id: 'COVID_WS_Address_Contacts_Of_Suspected_And_Confirmed_Cases_WS',
```

Comment:
Good call using a meaningful prefix for the id! I guess `WS` is not required again in the end

```suggestion
  id: 'COVID_WS_Address_Contacts_Of_Suspected_And_Confirmed_Cases',
```